test/powershell/Host/HostUtilities.Tests.ps1

```diff
@@ -1,26 +1,8 @@
-﻿function Get-RemoteRunspace {
-    $wc = [System.Management.Automation.Runspaces.WSManConnectionInfo]::new()
-
-    # Use AppVeyor credentials if running in AppVeyor, rather than implicit credentials.
-    try
-    {
-	    $appveyorRemoteCredential = Import-Clixml -Path "$env:TEMP\AppVeyorRemoteCred.xml"
-    }
-    catch { }
-    if ($appveyorRemoteCredential)
-    {
-        Write-Verbose "Using global AppVeyor credential";
-        $wc.Credential = $appveyorRemoteCredential
-    }
-    else
-    {
-        Write-Verbose "Using implicit credentials"
-    }
-
-    $remoteRunspace = [runspacefactory]::CreateRunspace($host, $wc)
-    $remoteRunspace.Open()
-
-    return $remoteRunspace
+﻿if (-not (Get-Module TestRemoting -ErrorAction SilentlyContinue))
+{
+    $root = git rev-parse --show-toplevel
```

Comment:
I don't think we should rely on tests being in a `git` tree, e.g. test passes in the lab might just copy our test tree.

Instead, I think we should use a path relative to `$PSScriptRoot`.